### ISSUE TITLE
[🐛 Bug]:  Personnals results don't wwork

### ISSUE THREAD

**Issue by wouf336** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#821]
### What went wrong?

i cant ativate personnals results on my android phone
(i have google beta)

### What did you expect to happen?

i can't activate personnals results on mobile google assistant app.

### Steps to reproduce the bug

Go to allow notification on your android device
bug

### App Version

v1.1.0

### OS Version

Windows 11/Android 12

### Relevant Assets

https://user-images.githubusercontent.com/52755262/202208933-c56032b0-124e-491c-ad49-5158c6f3ac9d.mp4

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by harisnadeem** [comment]
Yep, this issue is still there. I think this is something at Google's end to fix.

**Comment by im-coder-lg** [comment]
What we found was a loophole in the SDK. Maybe Google restricted this to select companies?
